en/docs/deploy/security/create-new-keystores.md

```diff
@@ -15,20 +15,20 @@ Let's get started with creating new keystores.
 There are two ways to create keystores for WSO2 Identity Server. You can either generate a keystore using an already existing public key certificate (CA-signed), or you can create the public key certificate at the time of generating the keystore. See the instructions given below.
 
 !!! note "Important"
-    If you are creating a new keystore for [data encryption](../../../deploy/security/encrypt-passwords-with-cipher-tool), make sure to acquire a public key certificate that contains the **Data Encipherment** key usage as explained in [recommendations for setting up keystores](../../../deploy/security/use-asymmetric-encryption#recommendations-for-setting-up-keystores). Otherwise, the following error can occur when you attempt data encryption:
+    If you are creating a new keystore for [data encryption](../../../deploy/security/encrypt-passwords-with-cipher-tool), make sure to acquire a public key certificate that contains the **Data Encipherment** key usage as explained in [recommendations for setting up keystores](../../../deploy/security/use-asymmetric-encryption#recommendations-for-setting-up-keystores). Otherwise, the following error can occur when you attempt data encryption.
     
-    ``` java tab="Error"
+    ``` java
     Exception in thread "main" org.wso2.ciphertool.CipherToolException: Error initializing Cipher at org.wso2.ciphertool.CipherTool.handleException(CipherTool.java:861) at org.wso2.ciphertool.CipherTool.initCipher(CipherTool.java:202) at org.wso2.ciphertool.CipherTool.main(CipherTool.java:80) Caused by: java.security.InvalidKeyException: Wrong key usage at javax.crypto.Cipher.init(DashoA13..) at javax.crypto.Cipher.init(DashoA13..) at org.wso2.ciphertool.CipherTool.initCipher(CipherTool.java:200)... 1 more
     ```
     
 
 ### Create a keystore using an existing certificate
 
-Secure Sockets Layer (SSL) is a protocol that is used to secure communication between systems. This protocol uses a public key, a private key and a random symmetric key to encrypt data. As SSL is widely
+Secure Sockets Layer (SSL) is a protocol that is used to secure communication between systems. This protocol uses a public key, a private key, and a random symmetric key to encrypt data. As SSL is widely
 used in many systems, certificates may already exist that can be reused. In such situations, you can use an already existing CA-signed certificate to generate your keystore for SSL by using OpenSSL and Java
 keytool.
 
-1.  First, you must export certificates to the **[PKCS12/PFX](http://en.wikipedia.org/wiki/PKCS_12)** format. 
+1.  Export certificates to the **[PKCS12/PFX** format. 
```

Comment:
why was this link removed?

Reply:
We do not point to wikipedia information as a practice due to chances of inaccuracy.